sdk/servicebus/service-bus/src/constructorHelpers.ts

```diff
@@ -43,9 +42,9 @@ export interface ServiceBusClientOptions {
 /**
  * @internal
  *
- * @param {string} connectionString
- * @param {(SharedKeyCredential | TokenCredential)} credential
- * @param {ServiceBusClientOptions} options
+ * @param connectionString -
+ * @param credential -
+ * @param options -
```

Comment:
If we have nothing nice to say about these parameters, we can remove the `@param` tags altogether. Here and in many other places in this PR

Reply:
I had nothing nice to say about this, so I said nothing at all (removed)